packages/react/src/components/TextArea/TextArea.js

```diff
@@ -13,19 +13,22 @@ import { WarningFilled16 } from '@carbon/icons-react';
 
 const { prefix } = settings;
 
-const TextArea = ({
-  className,
-  id,
-  labelText,
-  hideLabel,
-  onChange,
-  onClick,
-  invalid,
-  invalidText,
-  helperText,
-  light,
-  ...other
-}) => {
+const TextArea = React.forwardRef(function  TextArea(
```

Comment:
not sure why prettier/the linter didn't catch this

```suggestion
const TextArea = React.forwardRef(function TextArea(
```